doc/en/user/source/rest/api/featuretypes.rst

```diff
@@ -155,6 +155,20 @@ The ``recalculate`` parameter specifies whether to recalculate any bounding boxe
 * ``recalculate=nativebbox``: Recalculate the native bounding box, but do not recalculate the lat/long bounding box.
 * ``recalculate=nativebbox,latlonbbox``: Recalculate both the native bounding box and the lat/long bounding box.
 
+``Projection Policy``
+^^^^^^^^^^^^^^^^^^^^^
+.. list-table::
```

Comment:
Please add some context to this table so that the user can cleary understand understand the meaning of it
```suggestion
   When specifying the Projection Policy in a FeatureType defined in the request body, the internal name should be used instead of the one available on the UI. The following table shows the correspondence between display and internal names:
.. list-table::
```